### DISCUSSION TITLE
TX16S RS Slider and Throttle conflict in betaflight, edgetx 2.7.1

### DISCUSSION THREAD

**Discussion by astrobond** [EdgeTX/edgetx#2555] [Q&A]
Hi all :)
I found an issue with my TX16S MKII:
When i want to arm my quads and the RS Slider is a little of centered, the quad won't arm...
On the TX16S screen, the throttle channel is at zero, not moving, but in betaflight the throttle channel is moving when i move the RS Slider !!!???
There is no mixing activated.
Does someone have the same issue and a solution ?
Btw, no problem to arm when the RS Slider is centered.
Best regards, Astrobond.

**Comment by pfeerick** [comment]
So when you're at the channel monitor, you don't see the throttle channel move at all when you move the RS (right slider)? As that sounds very strange, and haven't heard of that before, nor have I noticed it with my own TX16Ss or TX16S-MKII. That still somewhat sounds like some sort of mixing somehow.

**Reply by astrobond** [reply]
@pfeerick , yes, when moving RS right slider, only RS channel (CH14) is moving in the channel monitor (outputs and mixers), nothing moving in the throttle channel (CH03) monitor... Could this be a hardware problem ?

**Reply by pfeerick** [reply]
>  when moving RS right slider, only RS channel (CH14) is moving in the channel monitor (outputs and mixers), nothing moving in the throttle channel (CH03) monitor.

Ok.... yet BF *is* showing CH03 moving (as well as CH14?). And when you move the throttle stick, CH03 moves, but not CH14?

> Could this be a hardware problem?

I wouldn't think so at the moment - the channel monitor should match what BF is seeing. So if the channel monitor is saying one thing, and BF is saying another... this sounds more like a software issue. Hardware issue would be more like a grounding issue making two inputs move strangely, wrong inputs moving (but again, still visible on the radio via things like channel monitor).

**Reply by astrobond** [reply]
In BF, (dsmx receiver) i don't see any other channels moving (just the throttle one) when moving the RS slider (same with LS btw, nothing moving at all) and the other channels are not moving when i move the throttle stick.
BTW, the throttle channel only moves a little up to 1150 and then down to 990 when i lower the RS slider from the center position, nothing happends when i go up with the RS slider... very strange :)
Another thing i forgot to say: when then RS slider is in a position that send a modified value to the throttle channel in BF, moving the throttle stick does nothing in BF... for example, when RS is in the low position, BF says that Throttle channel is at 990, and moving my throttle stick changes nothing in BF, locked at 990 until i put the RS slider in the center position, or upper positions.
Same problem with multiples quads and SPM4648 and SPM4650 receivers.

**Reply by pfeerick** [reply]
Oh, ok, that is sounding a little like hardware then... 

Try going into radio settings -> hardware,  and at the bottom go into into analogs. 

Have a look at what happens with `02` (throttle) and `13` (RS). It's normal to get like a 1/2 "jitter" even without any stick movement, but if you are seeing `02` (throttle) change dramatically if you are moving `13` (RS) you may need to reseat the connectors on the mainboard.

**Reply by astrobond** [reply]
@pfeerick thank you for helping :) just tested and no problem in the hardware analog debug, no "mixing" here between 02 and 13.
